src/StreamService/appsettings.json

```diff
@@ -0,0 +1,10 @@
+{
+  "Logging": {

```

Comment:
are you sure this file should look like this?

Reply:
fixed